advanced/readme_kiali.adoc

```diff
@@ -0,0 +1,183 @@
+
+----
+NAME                                        READY     STATUS      RESTARTS   AGE
+elasticsearch-0                             0/1       Running     0          1d
+grafana-6bb556d859-sr2kn                    0/1       Running     0          1d
+istio-citadel-d8b87b7fb-m6kr8               0/1       Running     0          1d
+istio-egressgateway-5c7847547d-l79bc        0/1       Running     0          1d
+istio-ingress-d4898f4d8-c45kz               0/1       Running     0          1d
+istio-ingressgateway-54ddb65d74-sznf9       0/1       Running     0          1d
+istio-pilot-58c7c644c7-l88d7                0/2       Running     0          1d
+istio-policy-586559fdd9-l88v9               0/2       Running     0          1d
+istio-sidecar-injector-6fc85fbd55-6r2hj     0/1       Running     0          1d
+istio-statsd-prom-bridge-6dbb7dcc7f-7qmbw   0/1       Running     0          1d
+istio-telemetry-7988677557-tz4sm            0/2       Running     0          1d
+jaeger-agent-lmhbb                          0/1       Running     0          0s
+jaeger-collector-68fd846775-wll4r           0/1       Running     0          1d
+jaeger-query-58f4655965-q8jjk               0/1       Running     0          1d
+kiali-7966d98797-77gqz                      0/1       Running     0          1d
+prometheus-586d95b8d9-r6d6x                 0/1       Running     0          1d
+----
+
+If the "STATUS" column for Kiali is "Running", you can check which URL it is
+using by using this command:
+
+[source, bash]
+----
+NAME                   HOST/PORT                                                PATH      SERVICES               PORT              TERMINATION   WILDCARD
+grafana                grafana-istio-system.192.168.15.12.nip.io                          grafana                http                            None
+istio-ingress          istio-ingress-istio-system.192.168.15.12.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.192.168.15.12.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.192.168.15.12.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.192.168.15.12.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.192.168.15.12.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.192.168.15.12.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (in my case,
+"kiali-istio-system.192.168.15.12.nip.io"), so let's access it:
+
+image::https://i.imgur.com/rG5tdZu.png[Login Page]
+
+This is the login page. The default credentials is "admin/admin", but it's
```

Comment:
The default credentials are "admin/admin".  You should change the default password before using Kiali in production.